test/Mono.Linker.Tests.Cases.Expectations/Support/DynamicallyAccessedMembersAttribute.cs

```diff
@@ -2,34 +2,132 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Runtime.CompilerServices
+namespace System.Diagnostics.CodeAnalysis
 {
+	/// <summary>
+	/// Specifies the types of members that are dynamically accessed.
+	///
+	/// This enumeration has a <see cref="FlagsAttribute"/> attribute that allows a
+	/// bitwise combination of its member values.
+	/// </summary>
+	/// <remarks>
+	/// This is a copy of the enum definition in the framework at https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMemberTypes.cs
+	/// </remarks>
 	[Flags]
-	public enum DynamicallyAccessedMemberKinds
+	public enum DynamicallyAccessedMemberTypes
```

Comment:
Please move this type to a new file and create new folder called System.Diagnostics.CodeAnalysis